### DISCUSSION TITLE
Flashing doesn’t work

### DISCUSSION THREAD

**Discussion by Bakuku** [darthcloud/BlueRetro#444] [Q&A]
For some reason I can’t flash the ESP32.

The tool just stays idle.
![A2AA4C2B-20E0-474C-B27C-E5E23E9DA178](https://user-images.githubusercontent.com/23058894/197756530-4e930e22-5b3c-4069-8538-4351481be635.jpeg)
![011F17F8-C3DB-4449-900E-E66990455BF4](https://user-images.githubusercontent.com/23058894/197756536-a7e268a5-434b-4e8e-ad51-88ee360ed7fa.jpeg)

ESP32_devkitc_v4

**Comment by stefanpausch** [comment]
You have to install the "CP210x Windows Driver" from https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers?tab=downloads and check your Windows device manager for the correct com port :)

**Reply by Bakuku** [reply]
I’ve done that already but still thanks!

For now it seems to work on my laptop but not my desktop pc.
